lib/lightning_web/live/run_live/index.html.heex

```diff
@@ -720,6 +720,7 @@
                 page={@page}
                 async_page={@async_page}
                 url={@pagination_path}
+                help_text="Use whole words or the start of words for better results; partial string matching often won't find substrings in the middle or at the end of a word."
```

Comment:
added to make it easier for users to understand the limitations of this `tsquery`